test/rauc.t

```diff
@@ -132,7 +137,7 @@ grep -q "ENABLE_SERVICE 1" $SHARNESS_BUILD_DIRECTORY/config.h &&
   select_system_or_session_bus
 
 # Prerequisite: openssl available [OPENSSL]
-openssl asn1parse -help &&
+openssl asn1parse -help 2>/dev/null &&
```

Comment:
This and the other changes are unrelated to the commit they come with. Should be split into a separate commit and note why it's needed.

Reply:
It is not needed and i can remove it again.